src/components/users/users-table-groups.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { FC } from "react";
+import { MetaformMember, MetaformMemberGroup } from "generated/client";
+import { Box, Chip, FormControl, MenuItem, OutlinedInput } from "@mui/material";
+import Select, { SelectChangeEvent } from "@mui/material/Select";
+
+/**
+ * Component props
+ */
+interface Props {
+  metaformMember: MetaformMember;
+  metaformMemberGroups: MetaformMemberGroup[];
+  onMetaformGroupMembershipAdd: (metaformMember: MetaformMember, groupId: string) => void;
+  onMetaformGroupMembershipRemove: (metaformMember: MetaformMember, groupId: string) => void;
+}
+
+/**
+ * Users table
+*/
+const UsersTableGroups: FC<Props> = ({ metaformMember, metaformMemberGroups, onMetaformGroupMembershipAdd, onMetaformGroupMembershipRemove }: Props) => {
+  const memberId = metaformMember.id!!;
+  const selectedGroups = metaformMemberGroups.filter(metaformMemberGroup => metaformMemberGroup.memberIds.includes(memberId));
+  const selectedGroupIds = selectedGroups.map(selectedGroup => selectedGroup.id!!);
+
+  /**
+   * Handle change event listener
+   * 
+   * @param event event
+   */
+  const handleChange = (event: SelectChangeEvent<string[]>) => {
```

Comment:
Method name could be more describing. handleMemberGroupSelectChange for example.